go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -0,0 +1,173 @@
+package vtgate
```

Comment:
Not yet, it's more of a preview. I was going to add them later, it's on my list.

First, I need to make the users and passwords configurable.